distributed/config.py

```diff
@@ -25,6 +25,7 @@
     'allowed-failures': 'distributed.scheduler.allowed-failures',
     'bandwidth': 'distributed.scheduler.bandwidth',
     'default-data-size': 'distributed.scheduler.default-data-size',
+    'events-cleanup-delay': 'distributed.scheduler.events-cleanup-delay',
```

Comment:
This dictionary is intended for backwards compatibility.  I don't think that we need this alias.  I would prefer that users today use the fully qualified name.